packages/lib-classifier/src/store/FeedbackStore.js

```diff
@@ -0,0 +1,73 @@
+import { autorun } from 'mobx'
+import { addDisposer, getRoot, types } from 'mobx-state-tree'
+import _ from 'lodash'
+
+import * as helpers from '../features/feedback/shared/helpers'
+import strategies from '../features/feedback/shared/strategies'
+
+const FeedbackStore = types
+  .model('FeedbackStore', {
+    active: types.optional(types.boolean, false),
```

Comment:
While the PFE implementation used `active`, the term is usually used in the FEM when referring to an instance of a resource, so maybe I want to name this boolean attribute something different?

Reply:
That's a good idea - since it's a boolean, something like `feedbackIsActive` maybe?

Reply:
`isActive` because `feedback.isActive` reads better than `feedback.feedbackIsActive`?